monitoring_service/server.py

```diff
@@ -81,7 +82,9 @@ def __init__(
         if not is_same_address(state_db.monitoring_contract_address(), monitor_contract_address):
             raise StateDBInvalid("Monitoring contract address doesn't match!")
         self.task_list: List[gevent.Greenlet] = []
-        if not is_service_registered(self.blockchain.web3, monitor_contract_address, self.address):
+        if not is_service_registered(
+            self.blockchain.web3, contract_manager, monitor_contract_address, self.address
```

Comment:
Current style guide is to put each argument on a new line.

Reply:
Done, but I don't see this explicitly in the style guide. If I didn't just miss it, we might want to add it. https://github.com/karlb/raiden/blob/master/CONTRIBUTING.md#coding-style

Reply:
You're right, but we do this everywhere in Raiden. If you want you can update the styleguide, but in Malta I got the feeling that we might switch to `black` eventually, so might not be necessary.